api/http/dataplane/data-plane-api.yaml

```
@@ -145,26 +145,38 @@ components:
     ConfigurationStatus:
       type: object
       properties:
+        instanceId:
+          type: string
+          example: "aecea348-62c1-4e3d-b848-6d6cdeb1cb9c"
         correlationId:
           type: string
           example: "9e2d49c9-ada2-4ed1-b6f2-391d5cf634d9"
-        lastUpdated:
+        events:
+          type: array
+          items:
+            $ref: "#/components/schemas/Event"
+    Event:
+      type: object
+      properties:
```

Should we make some of these fields required (so they aren't pointers in our generated code)?
```suggestion
    Event:
      type: object
      required:
        - timestamp
        - status
        - message
      properties:
```

---

Ya probably should make some things required. Not much in api is required at the moment so should we also discuss what should be required in the rest of the api ?

---

I don't mind making the changes here to make the fields required, but ya might be best to discuss what needs to be required if all of them don't need to be required and do a follow up PR